evansmusomi/react-native-102
# Updates README with Missing Features

*What this does*
- Adds missing features to the readme

*How to verify*
- When you visit the readme, you should see a complete list of features implemented
